site/content/docs/browser-checks/_index.md

```diff
@@ -139,6 +139,12 @@ Valid Playwright Test or Playwright scripts are the foundation of a valid Browse
 If the script fails, your check fails.
 {{< /info >}}
 
+### Browser check templates
+
```

Comment:
moved the templates explanation to the getting started page

Reply:
Good idea!